learn/guides/distinctive-language-features/data.md

```diff
@@ -68,7 +69,7 @@ The second one is the ``xml`` type. It is a sequence type built on a sequence of
 
 One of the crucial features of plain data is that it can be made immutable. You cannot do that for objects. Plain data consisting of simple and string values are inherently immutable.
 
-Structural values can be constructed either as mutable or immutable. The value includes a flag that indicates whether it's immutable or not and it is fixed at the time of construction of the value. Attempting to mutate an immutable structure causes a panic at runtime.
+Structural values can be constructed either as mutable or immutable. The value includes a flag that indicates whether it's immutable or not and it is fixed at the time of construction of the value. Attempting to mutate an immutable structure causes a `panic` at runtime.
```

Comment:
This is a description of the behaviour rather than a reference to the panic statement, right? So I don't think we should emphasize panic here.

Reply:
Causes a panic; doesn't it mean an actual panic rather. Since we are explaining the runtime behaviour here. If we are to use non-technical word, it should be causes an error at runtime IMO.

Reply:
I didn't mean a non-Ballerina word. Usually we emphasize keywords depending on the context. For example, we wouldn't emphasize "error" wherever it is used, unless we are referring to the `error` type. Similarly emphasized panic would ideally refer to the panic statement.

We use "result in a panic", "will panic", etc., in the spec also. So IMO the current version is OK.

Reply:
Suggestion addressed in https://github.com/ballerina-platform/ballerina-dev-website/pull/3689